.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 
 /bazel-*
+MODULE.bazel.lock
```

Comment:
This will include the `examples/*/MODULE.bazel.lock` as well, correct?

Reply:
Yep!